src/core/Form/SearchInput/SearchInput.baseStyles.tsx

```diff
@@ -54,6 +54,8 @@ export const baseStyles = withSuomifiTheme(
         margin-bottom: 1px;
         ::placeholder {
           font-style: italic;
+          color: ${theme.colors.depthDark2};
+          opacity: 1;
```

Comment:
Why is opacity needed? Is there some transparency set as default?

Reply:
Yeah, by default the lighter shade comes from opacity. Opacity 1 is needed to get the actual color in the designs.

Reply:
Probably just missing the '&'. Works both ways. Probably ended up like this by accident since the values below had this syntax. I'll add the '&' though.